holoviews/element/util.py

```diff
@@ -134,7 +139,13 @@ def _aggregate_dataset(self, obj, xcoords, ycoords):
         dtype = 'dataframe' if pd else 'dictionary'
         dense_data = Dataset(data, kdims=obj.kdims, vdims=obj.vdims, datatype=[dtype])
         concat_data = obj.interface.concatenate([dense_data, obj], datatype=[dtype])
-        agg = concat_data.reindex([xdim, ydim], vdims).aggregate([xdim, ydim], reduce_fn)
+        reindexed = concat_data.reindex([xdim, ydim], vdims)
+        if pd:
```

Comment:
Why not use ``reindexed.interface.dframe(dimensions=None, copy=False)`` instead of exposing the copy keyword argument at the element level? For ``copy=False`` to work, you are already assuming a dataframe type interface is being used...